cmd/earthly/main.go

```diff
@@ -627,6 +632,20 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 			Value:       defaultEnvFile,
 			Destination: &app.envFile,
 		},
+		&cli.StringFlag{
+			Name:        "secrets-file",
```

Comment:
This is very close to the already existing `--secret-file`. I wonder if we can think of a variation that isn't as close.

Reply:
that's a real good point. perhaps renaming them to `--earthlysecrets` (and `--earthlyargs`) would be more clear?

Reply:
Yeah, that makes sense I think.. although it's less consistent with kebab case being used in other args. But I think that it's better this way.